securing_apps/topics/client-registration/client-registration-cli.adoc

```diff
@@ -1,395 +1,360 @@
 
 
 [[_performing_crud_operations]]
-==== Creating client configuration
+==== Creating a client configuration
 
-After authenticating with credentials or configuring Initial Access Token, the first operation will usually be to create a new client.
+The first task after authenticating with credentials or configuring an Initial Access Token is usually to create a new client. Often you might want to use a prepared JSON file as a template and set or override some of the attributes.
 
-We've seen the simplest create command already. Often we may want to use a prepared JSON file as a template and set / override
-some of the attributes. For example, here is how you read a JSON file, override any `clientId` it may contain,
-set any other attributes as well, and after successful creation print the configuration to standard output.
+The following example shows how to read a JSON file, override any client ID it may contain, set any other attributes, and print the configuration to a standard output after successful creation.
 
-On Linux:
-[source,bash]
+* Linux:
+[options="nowrap"]
 ----
 $ kcreg.sh create -f client-template.json -s clientId=myclient -s baseUrl=/myclient -s 'redirectUris=["/myclient/*"]' -o
 ----
-
-On Windows:
-[source,bash]
+* Windows:
+[options="nowrap"]
 ----
 C:\> kcreg create -f client-template.json -s clientId=myclient -s baseUrl=/myclient -s "redirectUris=[\"/myclient/*\"]" -o
 ----
 
+Run the [command]`kcreg create --help` for more information about the [command]`kcreg create` command.
 
-See `kcreg create --help` for more information about `kcreg create`.
+You can use [command]`kcreg attrs` to list available attributes. Keep in mind that many configuration attributes are not checked for validity or consistency. It is up to you to specify proper values. Remember that you should not have any ID fields in your
+template and should not specify them as arguments to the [command]`kcreg create` command.
 
 
-You can use `kcreg attrs` to list available attributes. Keep in mind that many configuration attributes are not checked for
-validity or consistency. It is up to you to specify proper values. Also note that you should not have any `id` fields in your
-template and should not specify them as arguments to `kcreg create`.
+==== Retrieving a client configuration
 
+You can retrieve an existing client by using the [command]`kcreg get` command.
 
-==== Retrieving client configuration
+For example, on:
 
-You can retrieve an existing client by using `kcreg get`.
-
-For example, on Linux:
-[source,bash]
+* Linux:
+[options="nowrap"]
 ----
 $ kcreg.sh get myclient
 ----
-
-On Windows:
-[source,bash]
+* Windows:
+[options="nowrap"]
 ----
 C:\> kcreg get myclient
 ----
 
+You can also see the client configuration as an adapter configuration file, which you can package with your web application.
```

Comment:
Instead of 'You can also see' it would be better to say 'You can also retrieve'.